DolbyIO/Source/SdkAccessModule/Private/SdkAccess.cpp

```diff
@@ -146,6 +146,48 @@ namespace Dolby
 		Devices.Reset(MakeUnique<FDeviceManagement>(Sdk->device_management(), *Observer,
 		                                            [this](int Id) { return MakeHandler(Id); })
 		                  .Release());
+
+		Sdk->conference()
+		    .add_event_handler([this](const conference_status_updated& Event) { Status.SetStatus(Event.status); })
+		    .on_error(MakeHandler(__LINE__));
+
+		Sdk->conference()
+		    .add_event_handler(
+		        [this](const participant_added& Event)
+		        {
+			        ParticipantIDs.Add(Event.participant.user_id.c_str());
+			        Observer->OnListOfRemoteParticipantsChanged(ParticipantIDs);
+		        })
+		    .on_error(MakeHandler(__LINE__));
+
+		Sdk->conference()
+		    .add_event_handler(
+		        [this](const participant_updated& Event)
+		        {
+			        const auto& ParticipantStatus = Event.participant.status;
+			        if (ParticipantStatus)
+			        {
+				        if (*ParticipantStatus == participant_status::left)
+				        {
+					        ParticipantIDs.Remove(Event.participant.user_id.c_str());
+					        Observer->OnListOfRemoteParticipantsChanged(ParticipantIDs);
+				        }
+			        }
+		        })
+		    .on_error(MakeHandler(__LINE__));
+
+		Sdk->conference()
+		    .add_event_handler(
+		        [this](const active_speaker_change& Event)
+		        {
+			        FParticipants ActiveSpeakers;
+			        for (const auto& Speaker : Event.active_speakers)
+			        {
+				        ActiveSpeakers.Add(Speaker.c_str());
+			        }
+			        Observer->OnListOfActiveSpeakersChanged(ActiveSpeakers);
+		        })
+		    .on_error(MakeHandler(__LINE__));
```

Comment:
I didn't know this can work without a conference, but I guess it makes sense. I would also suggest to put this into an AddEventHandlers method.